src/server.h

```diff
@@ -2021,6 +2021,7 @@ struct redisServer {
     unsigned long long cluster_link_msg_queue_limit_bytes;  /* Memory usage limit on individual link msg queue */
     int cluster_drop_packet_filter; /* Debug config that allows tactically
                                    * dropping packets of a specific type */
+    int process_clustermsg_extensions; /* Debug config that allows disabling aux cluster message extensions feature. */                                  
```

Comment:
I know we don't have this practice yet but can we consider using a common prefix for all debug flags? such as `dbg_`?

also if we make the receiver agnostic to the extension and only use this flag to control the sender's behavior, we should probably consider a different name, such as `dbg_node_has_no_extension` or something along that line?

Reply:
Or perhaps put all debug flags in a sub-structure.

Reply:
A strong opinion from me, I want to stop adding them to the debug command and instead actually just build normal configs optionally for debug builds that aren't in the release build.

Reply:
`enable-debug-command` is a good enough guardrail at this moment and I feel it doesn't need to be addressed with this PR. I'm renaming it to `dbg_node_send_extensions` defaults to `1`. WDYT @madolson @PingXie ?

Reply:
@daniel-house Yeah, we could do that as well. Leaving it around the top level struct for now.